### ISSUE TITLE
expect('123').toBeNumericString();

### ISSUE THREAD

**Issue by JamieMason** [JamieMason/Jasmine-Matchers#39]
``` javascript
expect('123').toBeNumericString();
expect('1,234,567.00').toBeNumericString();
expect('   123  ').not.toBeNumericString();
expect('1..0').not.toBeNumericString();
```

**Comment by jacobwarduk** [comment]
What is the expected behaviour in the following cases?

1. `expect('1e100').toBeNumericString();`
2. `expect('1.0e+100').toBeNumericString();`
3. `expect('0x44c').toBeNumericString();`

**Comment by JamieMason** [comment]
hmm, I guess those should be covered as well.

**Comment by jacobwarduk** [comment]
Feature added in PR https://github.com/JamieMason/Jasmine-Matchers/pull/70

**Comment by JamieMason** [comment]
Sorry about this @jacobwarduk but similar to https://github.com/JamieMason/Jasmine-Matchers/issues/38#issuecomment-274784848, I've decided not to include this matcher (even though I created the issue in the first place).

Knowing more about it now, I worry there could be too many edge cases and because people depend on this library for their tests, I want it to be totally dependable so people can have confidence in it and their tests. This matcher might be a little risky.

Thanks a lot for your work on this Jacob, sorry to have wasted your time.
